opm/grid/cpgrid/CpGridData.hpp

```diff
@@ -141,7 +142,7 @@ template<class T, int i> struct Mover;
 class CpGridData
 {
     template<class T, int i> friend struct mover::Mover;
-
+    friend Dune::CartesianIndexMapper<CpGrid>;
```

Comment:
I think we can get rid of this friend declaration, too, if we use `chooseData()[level]->size(0)` to get the size.